src/Displayer.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react'
+import {CustomButtons} from './CustomButtons'
+
+export function Displayer(){
+   
+    const[firstName,setFirstName]= useState("");
+    const[lastName,setLastName]= useState("");
```

Comment:
Add some spacing here
```suggestion
    const [firstName, setFirstName]= useState("");
    const [lastName, setLastName]= useState("");
```